include/menus/scene_menu.h

```diff
@@ -0,0 +1,29 @@
+#include "menu.h"
+#include "libtww/d/kankyo/d_kankyo.h"
+#include "libtww/d/com/d_com_inf_game.h"
+#include "libtww/f_op/f_op_scene_req.h"
+#include "libtww/m_Do/m_Do_main.h"
+#include "controller.h"
+
+enum SceneMenuIndex {
+    MODIFY_WIND_INDEX,
+    TIME_DISP_INDEX,
+ //   DUNGEON_INDEX,
```

Comment:
Dungeon stuff is under flags, right? Then this should be deleted

Reply:
It should be